packages/loaders/src/TypingIndicator.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+import { isRtl } from '@zendeskgarden/react-theming';
+import { zdColorGrey600 } from '@zendeskgarden/css-variables';
+
+const COMPONENT_ID = 'loaders.spinner';
+
+const pulseAnimation = keyframes`
+  0% {
+    opacity: .2;
+  }
+
+  50% {
+    opacity: .8;
+  }
+
+  100% {
+    opacity: .2;
+  }
+`;
+
+const StyledCircle = styled.circle.attrs({
+  fill: 'currentColor',
+  cy: 2,
+  r: 2
+})``;
+
+const StyledTypingSvg = styled.svg.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  viewBox: '0 0 16 4',
+  width: ({ size }) => size,
+  height: ({ size }) => size * 0.25
+})`
+  color: ${({ color }) => color};
+
+  ${StyledCircle} {
+    opacity: 0.2;
+
+    &:nth-child(1) {
+      animation: ${pulseAnimation} 1s infinite;
+      animation-delay: ${props => (isRtl(props) ? 'unset' : '0.4s')};
+    }
+
+    &:nth-child(2) {
+      animation: ${pulseAnimation} 1s infinite;
+      animation-delay: 0.2s;
+    }
+
+    &:nth-child(3) {
+      animation: ${pulseAnimation} 1s infinite;
+      animation-delay: ${props => (isRtl(props) ? '0.4s' : 'unset')};
+    }
+  }
+`;
+
+/**
+ * All other props are spread onto the root `<svg>` element
+ */
+const TypingIndicator = ({ size, color, ...other }) => {
+  return (
+    <StyledTypingSvg size={size} color={color} {...other}>
+      <StyledCircle cx="14" />
+      <StyledCircle cx="8" />
+      <StyledCircle cx="2" />
+    </StyledTypingSvg>
+  );
+};
+
+TypingIndicator.propTypes = {
+  /* Width of the loader in px */
+  size: PropTypes.number,
+  color: PropTypes.string
+};
+
+TypingIndicator.defaultProps = {
+  size: 16,
+  color: zdColorGrey600
```

Comment:
Should this be `inherit` to match the other loaders?

Reply:
I didn't realizing that we were defaulting that with the other components.

@allisonacs we believe the for `Dots` it should be `blue-500` by default and for this inline loader, it should be `grey-600`, correct? If that's the case I think we should just default these otherwise we will have a wide variety of loading colors.

It's still customizable. Just with a better default.

Reply:
After talking with @allisonacs we decided to have a more opinionated default with this specific loader.  The `inherit` value can still be applied for the use-cases that require it.

Reply:
@austin94 @allisonacs we should've made the API consistent with other loaders. Let's revisit this minor detail at some point.